pkg/customflag/config.go

```
@@ -186,3 +187,28 @@ type s3ConfigFlag struct {
 	Bucket string
 	Folder string
 }
+
+type selinuxTestFlag bool
+
+func (s *selinuxTestFlag) Set(value string) error {
+	if value == "" {
```

if value arrives here empty , we should return an error right?

---

fixed

---

its not fixed lol

---

for some reason this change was not reflected in the PR but my git shows it was already added. so i had to add a new line after this section, to force add the change. hopefully the new error message is in the PR now.